Activities/Prac 5/emails.py

```diff
@@ -0,0 +1,33 @@
+"""
+CP1404 Practical 5
+Email list
+"""
+
+
+def main():
+    emails_and_names = {}
+    user_email = input("Email: ").lower()
```

Comment:
Do you think `.lower()` is needed in this program?

Reply:
it is more of a just in case. I have a cat play with my keyboard on a regular basis, never know if and when the caps lock is on.